packages/deployer/src/infrastructure/index.ts

```diff
@@ -78,11 +86,12 @@ async function deploy(airnodeIdShort: string, stage: string, region: string, con
   const options = { cwd: airnodeTmpDir };
   await runCommand(command, options);
 
+  const testingApiKeyVar = testingApiKey ? `-var='api_key=${testingApiKey}'` : '';
```

Comment:
You can think about adding some convenience functions to simplify creation of the command. I created similar ones for the admin CLI https://github.com/api3dao/airnode/blob/master/packages/admin/test/e2e/cli.feature.ts#L114

(Not in this PR)

Reply:
Created an [issue](https://github.com/api3dao/airnode/issues/461) for that.